package.json

```diff
@@ -6752,6 +6757,12 @@
 				"title": "Open Commit on Remote",
 				"icon": "$(globe)"
 			},
+			{
+				"command": "gitlens.views.openTagOnRemote",
+				"title": "Open Tag on Remote",
+				"category": "GitLens",
+				"icon": "$(globe)"
+			},
```

Comment:
Should also add `gitlens.views.openTagOnRemote.multi` command to follow the pattern of commit[s]

Reply:
added